woocommerce/woocommerce
# Adding the feedback button on activity panel for the classic product page

### Changes proposed in this Pull Request:



Closes #35806 .

### How to test the changes in this Pull Request:



1. Checkout branch.
2. Navigate to classic product editor. (Menu -> Products -> Add New)
3. Notice that the Feedback button appears in the header above:
![image](https://user-images.githubusercontent.com/444632/205164752-5fad8a5f-e354-45d0-a6b3-a7a18339cf64.png)

4. Click the feedback button.
5. Notice that the icon color changes, and the CES modal is displayed.
6. Open the console and enable tracks debugging with `localStorage.setItem( 'debug', 'wc-admin:*' );`
7. Click "Send" on the modal, and ensure that the `wcadmin_ces_feedback` event is sent with the `product_feedback` action and correct score(s).



### Other information:




### FOR PR REVIEWER ONLY:
